.github/workflows/test.yml

```diff
@@ -18,25 +18,22 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Create gcloud key file
-      run: echo ${{ secrets.GOOGLE_SERVICE_ACCOUNT_KEY }} > key.json
+      run: openssl base64 -d -A <<< '${{ secrets.GOOGLE_SERVICE_ACCOUNT_KEY }}' -out key.json
```

Comment:
Is it a more secure way to do the same `echo`?

Reply:
No, i had to change the key in secrets and had forgotten how i did it last time. So the value in secrets just wouldn't get echoed properly. With this the value in secrets is a  base64 encoded string. That's the only difference i guess, it's not more or less secret